src/main/kotlin/net/irisshaders/lilybot/utils/ResponseHelper.kt

```diff
@@ -3,13 +3,38 @@ package net.irisshaders.lilybot.utils
 import com.kotlindiscord.kord.extensions.DISCORD_BLACK
 import com.kotlindiscord.kord.extensions.DISCORD_RED
 import dev.kord.common.Color
+import dev.kord.core.behavior.channel.MessageChannelBehavior
 import dev.kord.core.behavior.channel.GuildMessageChannelBehavior
 import dev.kord.core.behavior.channel.createEmbed
 import dev.kord.core.entity.User
+import dev.kord.core.entity.Message
 import dev.kord.rest.builder.message.EmbedBuilder
 import kotlinx.datetime.Clock
 
 object ResponseHelper {
+    /**
+     * Creates an [EmbedBuilder] and populates it with the current timestamp, the given as the
+     * requester and the given [Color] as its color.
+     *
+     * @param embedTitle The title to set the embed to, or null to set no title.
+     * @param embedDescription The description of the embed. If null, the description will be empty.
+     * @param embedColor The color to set the embed to. If null, the embed will be coloured black.
+     * @author Maximumpower55
+     */
+    suspend fun responseEmbedInChannel(channel: MessageChannelBehavior, embedTitle: String?, embedDescription: String?, embedColor: Color?, requestedBy: User?): Message {
```

Comment:
The is the same as `responseEmbedInActionLog`, except i forgot to rename it.

Reply:
This too

Reply:
It actually isn't the same i cant use responseEmbedInActionLog because it doesn't return a message and the channel parameter is GuildMessageChannelBehavior instead of MessageChannelBehavior

Reply:
Oh funny.